docs/API.md

```diff
@@ -604,9 +613,93 @@ for event in events:
     print event
 ```
 
+<a name="get_bucket_encryption"></a>
+
+### get_bucket_encryption(bucket_name)
+
+Get default encryption configuration set on a bucket.
+
+__Parameters__
+
+|Param   |Type   |Description   |
+|:---|:---|:---|
+|``bucket_name``   | _string_  |Name of the bucket.|
+
+__Return Value__
+
+|Param   |Type   |Description   |
+|:---|:---|:---|
+|``encryption_configuration``    |  _dict_ | Default encryption configuration in dictionary format.|
+
+__Example__
+
+```py
+# Get the default encryption configuration set on bucket, "mybucket".
+encryption = minioClient.get_bucket_encryption('mybucket')
+print(encryption)
+```
+
+<a name="put_bucket_encryption"></a>
+
+### put_bucket_encryption(bucket_name, encryption_configuration)
+
+Put/Set default encryption configuration for a specified bucket.
+
+__Parameters__
+
+|Param   |Type   |Description   |
+|:---|:---|:---|
+|``bucket_name``  | _string_  |Name of the bucket.|
+|``encryption_configuration``   | _dict_ | Default encryption configuration in dictionary format.|
+
+__Example__
+
+```py
+# Sample default encryption configuration
+ENC_CONFIG = {
+    'ServerSideEncryptionConfiguration':{
+        'Rule': [
+            {'ApplyServerSideEncryptionByDefault': {'SSEAlgorithm': 'AES256'}}
+        ]
+    }
+}
+try:
+    minioClient.put_bucket_encryption('mybucket', ENC_CONFIG)
+except ResponseError as err:
+    print(err)
+```
+
+<a name="delete_bucket_encryption"></a>
+
+### remove default encryption configuration on a bucket
```

Comment:
This needs to be consistent with others titles

Reply:
ok